roles/metrics/README.md

```diff
@@ -0,0 +1,25 @@
+redhat.satellite_operations.metrics
+=============================
+
+Setup Performace Co-Pilot and Grafana to gather metrics of a Satellite installation.
```

Comment:
This probably needs to be sent upstream too:
```suggestion
Setup Performance Co-Pilot and Grafana to gather metrics of a Satellite installation.
```

Reply:
https://github.com/theforeman/foreman-operations-collection/pull/156